source/geometry_model/box.cc

```diff
@@ -318,6 +380,15 @@ namespace aspect
                                    "indicators 0 through 5 indicate left, right, front, back, bottom "
                                    "and top boundaries (see also the documentation of the deal.II class "
                                    "``GeometryInfo''). You can also use symbolic names ``left'', ``right'', "
-                                   "etc., to refer to these boundaries in input files.")
+                                   "etc., to refer to these boundaries in input files."
+                                   "It is possible to add a topography to the box model. Note however that "
+                                   "the representation of this topography depends greatly on the number "
+                                   "of repetitions specified for the coarse mesh, as the topography is not "
+                                   "updated in the initial global and adaptive refinement cycles.")
```

Comment:
Maybe also add a sentence that the topography is supposed to be small (because it is not taken into account to compute depth or representative point).